### PR TITLE
feat(logger): use iso format for buffer date

### PR DESCRIPTION
**Pull request by adamweeks**
https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-152946

Currently, the client logs that are uploaded are using the epoch time for indicating when the event was logged. Switch to iso format instead.

### CODE REVIEW

**Comment by arun3528** [comment]
thanks adam

File: packages/node_modules/@webex/plugin-logger/test/unit/spec/logger.js [hunk 1]
```diff
@@ -79,10 +79,19 @@ describe('plugin-logger', () => {
...
     });
 
+    it('adds the date to the beggining of the buffer entry', () => {
+      webex.logger.log('test date');
+
+      // Convert string back to date object
+      const logDate = new Date(webex.logger.buffer[0][0]);
+
+      // eslint-disable-next-line no-restricted-globals
+      assert.isTrue(logDate instanceof Date && !isNaN(logDate));
```
[the remark this answers is not in the document]
Comment (adamweeks) [reply packages/node_modules/@webex/plugin-logger/test/unit/spec/logger.js]: @mwegman updated

**Comment by glenatwork** [comment]
Ha I did basically the same thing here https://github.com/webex/webex-js-sdk/pull/1717 but didn't do a UT.  well done.

**Comment by adamweeks** [comment]
> Ha I did basically the same thing here #1717 but didn't do a UT. well done.

@glenatwork oh nice! I haven't had a chance to look over your PR, this was just a quick fix request from @arun3528

**Pull request merged**